### ISSUE TITLE
Problem trying to install aarch64 processor

### ISSUE THREAD

**Issue by jlbeltrana** [bigbluebutton/bigbluebutton#13723]
**Describe the bug**
When trying to install with the script bbb-install.sh on one of the new Oracle Cloud VMs with the Ampere A1 processor family it shows the following error:

bbb-install: You must run this command on a 64-bit server.

**Installation type**
bbb-install script

**Firewall and IP address type**
No network problems

**Console output**
command:
wget -qO- https://ubuntu.bigbluebutton.org/bbb-install.sh | bash -s -- -v bionic-230 -s bbb.dominio.com -e support@dominio.com -w -c turn.dominio.com:xxxxxxxxxxxxxxxxxxxxx

output:
You must run this command on a 64-bit server.

command:
uname -m

output:
aarch64

**Additional context**
Additional information about the new processor in Oracle Cloud:

https://community.arm.com/arm-community-blogs/b/operating-systems-blog/posts/oracle-cloud-infrastructure-a1-nginix?utm_source=oracle&utm_medium=partner&utm_campaign=2021_infrastructure-servers_mk03-2_na_na_na_awa&utm_content=blog

**Comment by mvasylenko** [comment]
arm64/aarch64 is not supported yet
https://github.com/bigbluebutton/bigbluebutton/issues/10677

**Comment by pushpamtrigyn** [comment]
Hello, Is arm64/aarch64 still not supported for setup ?

**Comment by dreamflasher** [comment]
Yeah, unfortunately still not supported despite all depencies now support it.

**Comment by toabi** [comment]
Minor updates to the install script and it also runs through on aarch64 server… BUT the issue is that the docker image is only amd64 :(

**Comment by dreamflasher** [comment]
@toabi could you create a PR of the install script that runs on aarch64? I believe that would be a great step for bbb. My impression was, when I tried to get it to run that it's not simple at all – including the need for cross compilation of dependencies. So if you found a way to get it to run, that would be a great contribution to the project. And then as soon as this is merged it should be simple to update the docker image.

**Comment by toabi** [comment]
Oops. I just noticed that I was just looking at and editing the greenlight-v3 stand-alone install script and not the whole BBB installer. Sorry for the wrong optimism here.

**Comment by Daniel-at-git** [comment]
IMHO the issue is still open. Maybe someone can add label:"type: enhancement".
